### PR TITLE
feat(Navigation): add label preview when icon not exist

### PR DESCRIPTION
**Pull request by ej9x**

### CODE REVIEW

File: src/components/Button/Button.js [hunk 1]
```diff
@@ -54,7 +55,7 @@ class Button extends Component<ButtonProps> {
         type={ type }
         loading={ loading }
         onClick={ this.onClick }
-        aria-busy={ String(loading) }
+        aria-busy={ String(!!loading) }
```
Comment (zouxuoz) [inline src/components/Button/Button.js]: @ej9x for what? We already have a boolean prop

Comment (vorobeez) [reply src/components/Button/Button.js]: Maybe `Boolean(loading).toString()`?

Comment (vorobeez) [reply src/components/Button/Button.js]: I think it's more reasonable and clear.

File: src/components/Navigation/NavigationItem.js [hunk 2]
```diff
@@ -19,7 +19,12 @@ type NavigationItemProps = {
 const NavigationItem = ({ icon, label, ...rest }: NavigationItemProps) => (
   <NavigationItemTag { ...rest }>
     <NavigationItemIcon modifiers={ rest }>
-      <Icon name={ icon || '' } color="WHITE" />
+      <If condition={ icon === undefined && typeof label === 'string' }>
```
Comment (vorobeez) [inline src/components/Navigation/NavigationItem.js]:
```
      <If condition={ icon === undefined && typeof label === 'string' && label.length > 0}>
        <NavigationItemLabelPreview>{ label.charAt(0).toUpperCase() }</NavigationItemLabelPreview>
      </If>
```

File: src/components/Button/Button.js (hunk 1 above)
Comment (andrei-anisimov) [reply src/components/Button/Button.js]: Saw this. What does 'aria' word mean?

Comment (zouxuoz) [reply src/components/Button/Button.js]: @andrei-anisimov attribute for detecting loading state of the button

Comment (andrei-anisimov) [reply src/components/Button/Button.js]: 'aria' - strange word. Did you mean 'area'?

Comment (vorobeez) [reply src/components/Button/Button.js]: @andrei-anisimov aria stands for "Accessible Rich Internet Applications". aria-* tags are used for accessibility.

Comment (andrei-anisimov) [reply src/components/Button/Button.js]: @vorobeez got you, thanks for the explanation!

**Pull request merged**